ntbs-integration-tests/NotificationPages/TreatmentEventsEditPageTests.cs

```diff
@@ -144,6 +160,32 @@ public async Task GetTreatmentEvents_RendersTreatmentTable_ForPopulatedEntries()
             Assert.Contains(TREATMENT_OUTCOME_SUBTYPE.GetDisplayName(), treatmentOutcomeTextContent);
         }
 
+        [Theory]
+        [InlineData(Utilities.DRAFT_WITH_DIAGNOSIS_DATE, TreatmentEventType.DiagnosisMade, "01 Jun 2021")]
+        [InlineData(Utilities.DRAFT_WITH_TREATMENT_START_DATE, TreatmentEventType.TreatmentStart, "15 May 2021")]
```

Comment:
I'd be tempted to add a test for when neither date has been set and there are no treatment events

Reply:
Yep, I agree! Done